spec/01-unit/08-router_spec.lua

```diff
@@ -1477,6 +1477,7 @@ describe("Router", function()
           },
         },
         -- regex
+        -- regex will no longer be normalized since 3.0
```

Comment:
duplicated comments, could we do some clean?

Reply:
The first is a title and the second is an explanation. Surely it is ambiguous.